log4j-to-slf4j/pom.xml

```diff
@@ -89,19 +89,5 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
-  <build>
-    <plugins>
-      <plugin>
-        <groupId>org.apache.felix</groupId>
-        <artifactId>maven-bundle-plugin</artifactId>
-        <configuration>
-          <instructions>
-            <Bundle-Activator>org.apache.logging.slf4j.Activator</Bundle-Activator>
-            <Export-Package>org.apache.logging.slf4j</Export-Package>
-            <Import-Package>org.slf4j*;version="${range;[==,${slf4j.support.bound})}",*</Import-Package>
```

Comment:
I would assume that this removal means, that the upper-bound for org.slf4j.* packages will be 2 (exclusive) again, instead of 3 (exclusive)?

Reply:
I am submitting fixes to the BND configuration one module at a time so that it can be more easily reviewed. The complete list will be in #1819.

I am keeping this conversation open until I'll check the SLF4J related modules.

Reply:
Sounds fine.
If you reach that module it is maybe sufficient to just add another `bnd`-configuration element for the bnd-maven-plugin with (CDATA) content like:
`Import-Package: org.slf4j*;version="${range;[==,${slf4j.support.bound})}",*`

This should overwrite the configuration from the parent: https://github.com/bndtools/bnd/tree/master/maven-plugins/bnd-maven-plugin#bnd-instruction-inheritance

And regarding other headers, https://github.com/bndtools/bnd/tree/master/maven-plugins/bnd-maven-plugin#reproducible-builds might be interesting as well (all just in case you didn't know that already #1142 ).